src/postgres-util/src/lib.rs

```diff
@@ -440,6 +440,8 @@ impl Config {
             config
                 .replication_mode(ReplicationMode::Logical)
                 .connect_timeout(Duration::from_secs(30))
+                .keepalives_interval(Duration::from_secs(60))
```

Comment:
I've seen it recommended when using both keepalives and tcp user timeouts to set `tcp_user_timeout = keepalive_idle + keepalive_count*keepalive_interval` so an unanswered socket is dropped at the same time regardless of whether the sender had pending messages.

e.g. if we want a 60s for tcp user timeout, maybe we want to give the other end 5 attempts to respond via keep alive, so we set keepalive idle = 10, keepalive interval = 10, keepalive count = 5 or something like that.

Reply:
ty for this config